### PR TITLE
issue13

### PR DESCRIPTION
**Pull request by thomasing**

### CODE REVIEW

File: examples/setrise/main.c [hunk 1]
```diff
@@ -37,10 +37,14 @@ double observer_next_sunset(const observer_t *observer, double time)
 //		printf("iteration %i: elev=%f\n", i++, sun.elevation*180.0/M_PI);
 	}
 
+	if (obs != NULL) {
```
Comment (ryeng) [inline examples/setrise/main.c]: Good idea!

File: examples/setrise/main.c [hunk 2]
```diff
@@ -5,7 +5,7 @@
 #include <predict/predict.h>
 #include <predict/observer.h>
 
-double observer_next_sunset(const observer_t *observer, double time)
+double observer_next_sunset(const observer_t *observer, double time, struct observation *obs)
```
Comment (ryeng) [inline examples/setrise/main.c:9]: Should this be a library function? It looks useful. Same for the other similar functions. Nice touch with setting obs to NULL if you just want the time.

File: include/predict/observer.h [hunk 3]
```diff
@@ -16,21 +16,21 @@ typedef struct observer {
...
-	double correctedElevation;         
-	///Range (km) 
-	double range;                        
-	///Range vector                    
-	double xRange, yRange, zRange; 
+	double time;
+	///Azimut angle (rad)
+	double azimut;
+	///Azimut angle rate (rad/s)
+	double azimut_rate;
```
Comment (ryeng) [inline include/predict/observer.h]: See issue #15.

File: examples/setrise/main.c (hunk 2 above)
Comment (thomasing) [inline examples/setrise/main.c:9]: Yes, plan was to move this function to the library when working correctly. I suggest we create a new issue for this feature and continue development in a new branch.

**Pull request merged**